src/serialisation.rs

```diff
@@ -28,16 +31,18 @@ pub enum SerialisationError {
 
 /// Function to serialise an Encodable type
 pub fn serialise<T>(data: &T) -> Result<Vec<u8>, SerialisationError>
-                                 where T: ::rustc_serialize::Encodable {
-    let mut encoder = ::cbor::Encoder::from_memory();
+    where T: Encodable
+{
+    let mut encoder = Encoder::from_memory();
     try!(encoder.encode(&[data]));
     Ok(encoder.into_bytes())
 }
 
 /// Function to deserialise a Decodable type
 pub fn deserialise<T>(data: &[u8]) -> Result<T, SerialisationError>
-                                      where T: ::rustc_serialize::Decodable {
-    let mut decoder = ::cbor::Decoder::from_bytes(data);
+    where T: Decodable
+{
```

Comment:
yes, rustfmt insist in opening it in a new line, even I manually enforced all the stuff into one line after removing full name qualification.